### PR TITLE
[UPDATES/ASSETS] Updated frontend dependencies and added an image optimizer

### PR DESCRIPTION
**Pull request by TheRedDaemon**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,6 +1,3 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
```
Comment (gynt) [inline .husky/pre-commit:2]: What is this?

File: package.json [hunk 2]
```diff
@@ -43,58 +43,60 @@
     "build": "tauri build",
     "lint": "eslint . --ext .ts,.tsx",
     "test": "vitest run",
-    "prepare": "husky install"
```
Comment (gynt) [inline package.json:46]: Is "install" not necessary anymore?

File: .husky/pre-commit (hunk 1 above)
Comment (TheRedDaemon) [reply .husky/pre-commit:2]: No idea, I just removed it. See https://github.com/typicode/husky/releases/tag/v9.0.1 under "How to migrate".

File: package.json (hunk 2 above)
Comment (TheRedDaemon) [reply package.json:46]: See other answer.

**Pull request merged**